recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -250,6 +250,16 @@ export default {
 .dt-feed-item-row {
   transition-duration: 2s !important;
 
+  &__left-time {
+    color: var(--dt-color-foreground-tertiary)!important;
+    padding-top: var(--dt-space-350)!important;
+    line-height: var(--dt-font-line-height-400)!important;
+    font-size: var(--dt-font-size-100)!important;
+    font-weight: var(--dt-font-weight-normal)!important;
+    white-space: nowrap!important;
+    height: 100% !important;
```

Comment:
Don't think we need !important here do we?

Reply:
updated